packages/fluentui/react-builder/src/config.tsx

```diff
@@ -0,0 +1,721 @@
+
+  Status: {
+    props: { state: 'success' } as FUI.StatusProps,
+  },
+
+  // Table: { props: { content: 'Table' } as FUI.TableProps },
+
+  Text: {
+    props: { content: 'Text' } as FUI.TextProps,
+  },
+
+  TextArea: {
+    props: { defaultValue: 'Hello there!' } as FUI.TextAreaProps,
+  },
+
+  // Toolbar: { props: { content: 'Toolbar' } as FUI.ToolbarProps },
+
+  // Tooltip: { props: { content: 'Tooltip' } as FUI.TooltipProps },
+
+  Tree: {
+    props: {
+      items: [
+        {
+          id: '1',
+          title: 'Level 1',
+          items: [
+            {
+              id: '11',
+              title: 'Level 1.1',
+              items: [
+                { id: '111', title: 'Level 1.1.1' },
+                { id: '112', title: 'Level 1.1.2' },
+                { id: '113', title: 'Level 1.1.3' },
+              ],
+            },
+            {
+              id: '12',
+              title: 'Level 1.2',
+              items: [
+                { id: '121', title: 'Level 1.2.1' },
+                { id: '122', title: 'Level 1.2.2' },
+                { id: '123', title: 'Level 1.2.3' },
+              ],
+            },
+          ],
+        },
+        {
+          id: '2',
+          title: 'Level 2',
+          items: [
+            {
+              id: '21',
+              title: 'Level 2.1',
+              items: [
+                { id: '211', title: 'Level 2.1.1' },
+                { id: '212', title: 'Level 2.1.2' },
+                { id: '213', title: 'Level 2.1.3' },
+              ],
+            },
+          ],
+        },
+      ],
+    } as FUI.TreeProps,
+  },
+
+  // Video: { props: { content: 'Video' } as FUI.VideoProps },
+};
+
+export const resolveComponent = (displayName): React.ElementType => {
+  return FUI[displayName] || FUIIcons[displayName] || displayName;
+};
+
+// FIXME: breaks for <button>btn</button>
+const toJSONTreeElement = input => {
+  if (isElement(input)) {
+    return {
+      $$typeof: 'Symbol(react.element)',
+      type:
+        typeof (input as React.ReactElement).type === 'string'
+          ? (input as React.ReactElement).type
+          : ((input as React.ReactElement).type as any).displayName,
+      props: toJSONTreeElement(input.props),
+    };
+  }
+  const result = _.transform(input, (acc, value, key) => {
+    if (Array.isArray(value)) {
+      acc[key] = toJSONTreeElement(value);
+    } else if (_.isPlainObject(value)) {
+      acc[key] = toJSONTreeElement(value);
+    } else {
+      acc[key] = value;
+    }
+  });
+  return result;
+};
+
+const getUUID = () =>
+  Math.random()
+    .toString(36)
+    .slice(2);
+
+export const resolveDraggingElement: (displayName: string, draggingElements?) => JSONTreeElement = (
+  displayName,
+  draggingElements = DRAGGING_ELEMENTS,
+) => {
+  const jsonTreeElement = toJSONTreeElement(draggingElements[displayName]);
+  return {
+    uuid: getUUID(),
+    $$typeof: 'Symbol(react.element)',
```

Comment:
```suggestion
    $$typeof: Element,
```

`Element` is exported from `react-is`:

![image](https://user-images.githubusercontent.com/14183168/84157483-15ab6c80-aa6b-11ea-89f1-cf3a6b2e9958.png)

Reply:
I cannot use `Element` as `symbol` - it does not get stringified in `JSON.stringify()` and there are also places where it is not usable (type definition). Let's keep it as is for now and discuss later.